File: src/com/l2kt/gameserver/network/clientpackets/RequestRecipeShopManageList.java
package com.l2kt.gameserver.network.clientpackets;

import com.l2kt.gameserver.model.actor.instance.Player;

public final class RequestRecipeShopManageList extends L2GameClientPacket
{
	@Override
	protected void readImpl()
	{
	}
	
	@Override
	protected void runImpl()
	{
		final Player player = getClient().getActiveChar();
		if (player == null)
			return;
		
		player.tryOpenWorkshop(true);
	}
}

File: src/com/l2kt/gameserver/network/clientpackets/RequestRecipeShopManageList.kt
package com.l2kt.gameserver.network.clientpackets

class RequestRecipeShopManageList : L2GameClientPacket() {
    override fun readImpl() {}

    override fun runImpl() {
        val player = client.activeChar ?: return

        player.tryOpenWorkshop(true)
    }
}